corehq/apps/domain/tests/test_domain_transfer.py

```diff
@@ -184,7 +184,8 @@ def test_permissions_for_transfer_domain_view(self):
         # No one logged in
         resp = self.client.get(reverse('transfer_domain_view',
                                        args=[self.domain.name]), follow=True)
-        self.assertEqual(resp.status_code, 404)
+        self.assertRedirects(resp, reverse('domain_login', kwargs={'domain': self.domain.name}) +
+            '?next=' + reverse('transfer_domain_view', args=[self.domain.name]))
```

Comment:
Yeah!

If check for `resp.status_code` with and without `follow` it will be `200` and `302`. But I think `assertRedirects` takes care of both responses.